go-client/session/radmin_rpc_types.go

```diff
@@ -15,7 +15,7 @@
 * KIND, either express or implied.  See the License for the
 * specific language governing permissions and limitations
 * under the License.
```

Comment:
Whether all of these lines should also be indented by one space ?

Reply:
It is generated by go-client/generator/main.go, I've updated the files.